question28/js/password.js

```diff
@@ -0,0 +1,35 @@
+import { validateInputValue } from "./modules/validations";
+
+const submitButton = document.getElementById('js-submit-button');
+const passwordInputArea = document.querySelector(".js-password");
+const confirmPasswordInputArea = document.querySelector(".js-confirm-password");
+
+
+const checkAllValidity = () => {
+  return document.getElementsByTagName("input").length === document.getElementsByClassName("valid").length && passwordInputArea.value === confirmPasswordInputArea.value;
+}
```

Comment:
③password.js
On this page, validation passes if the input values are consistent, in addition to if the valid class is satisfied.